mobile/src/api/images.queries.ts

```diff
@@ -10,7 +10,7 @@ export const useImages = () => {
       for (let index = 0; index < 50; index++) {
         try {
           const data = await axios.get(
-            'http://141.94.127.211:8000/get_unlabeled_random_event',
```

Comment:
I can have a look later but here we are doing something that is not great : we are awaiting for request `i` to finish before triggering request `i+1`.
A better approach would be to run all requests in parallel with `Promise.allSettled`. `Promise.allSettled` is better in this case than `Promise.all` because it won't throw an error if one of the call fails.

Reply:
Overall the best approach would be to have a route that returns a list of events, so that we don't need to trigger `n` calls to get `n` events.

Reply:
I am kind of react/tsx noob to implement Promise.all/allSettled but you are right, we want to avoid error if one of the call fails !

Yes, i agree -> a route returning a list of events (with a param `n` and a default value !) -> I am taking care to have this route soon !